test/functionalTest/cases/2507_csub_metadata_field/metadata_filter_existing_sub.test.DISABLED

```diff
@@ -0,0 +1,259 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
I haven't been able to finish this .test, due to some problems in update subscription logic to be solved in a very next PR. Thus, the .test file is added but in .DISABLED mode.

NTC (just informative).